src/Spain/Mews.Fiscalizations.Spain/Model/Request/Header.cs

```diff
@@ -4,13 +4,13 @@ namespace Mews.Fiscalizations.Spain.Model.Request
 {
     public sealed class Header
     {
-        public Header(LocalCompany company, CommunicationType communicationType)
+        public Header(LocalCounterParty localCounterParty, CommunicationType communicationType)
```

Comment:
Maybe let's call it assignee, so that people understand what it is?

```suggestion
        public Header(LocalCounterParty documentAssignee, CommunicationType communicationType)
```

Reply:
If I'm not mistaken, I usually find that in the documentations they refer to it as counterparty and not "assignee" that's why I decided to call it counterparty.

but i am also ok with calling it assignee.